src/client/components/Checker.tsx

```diff
@@ -49,6 +49,19 @@ export const Checker: FC<CheckerProps> = ({ config }) => {
   const googleAnalytics = useGoogleAnalytics()
   const outcomes = useRef<HTMLDivElement | null>(null)
 
+  // conditionally toggle description label between center align (1 line) and left align (> 1 line)
+  const descriptionRef = useRef<HTMLParagraphElement>(null)
```

Comment:
Could we just set `maxWidth` to `100%` for subtitle with no `text-align`? If the text is less than one line, the width will be auto to fit the content. If it is more than 1 line, it will be width `100%`.

![image](https://user-images.githubusercontent.com/3666479/120995123-b1270880-c7b7-11eb-8602-73e950c26cd5.png)